core/domain/feedback_services.py

```diff
@@ -167,7 +167,7 @@ def create_message(
     msg.put()
 
     # Update the message count in the thread.
-    if thread.message_count is not None:
+    if thread.message_count:
```

Comment:
@lilithxxx, was there any specific reason for this change? we fall into [this issue](https://github.com/oppia/oppia/pull/7206#issue-300305313) because of this and it was about to slip into the production.